<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindowB</class>
 <widget class="QMainWindow" name="ClientWindowB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindowB</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ClientWindowB {
	border-image: url(:/background/bg.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 15px;
	background-color: rgb(27, 154, 247);
    border-color: rgb(27, 154, 247);
    color: #FFF;
    font-size: 28px;
    height: 40px;
    line-height: 40px;
    padding: 0 30px;
	font-weight: 400;
	font-family: &quot;华文行楷&quot;;
	text-decoration: none;
	text-align: center;
	margin: 5px;
}
QPushButton:visited {
    color: #FFF;
}
QPushButton:hover, QPushButton:focus {
    background-color: #4cb0f9;
    border-color: #4cb0f9;
    color: #FFF;
	text-decoration: none;
    outline: none;
}

QPushButton.disabled, QPushButton.is-disabled, QPushButton:disabled {
    top: 0 !important;
    background: #EEE !important;
    border: 1px solid #DDD !important;
    color: #CCC !important;
    opacity: .8 !important;
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="title">
          <property name="text">
           <string>&lt;font style='color:white;font-size:70px;font-weight:700;font-family:&quot;汉呈王天喜榜书&quot;;'&gt;欢乐斗地主&lt;/font&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="beginButton">
            <property name="text">
             <string>开始连接</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="connectIP"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="connectIP">
   <property name="text">
    <string>连接IP</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
